core/domain/rating_services.py

```diff
@@ -84,9 +89,16 @@ def _update_user_rating():
 
 
 def get_user_specific_rating_for_exploration(user_id, exploration_id):
-    """
+    """Fetches a rating for the specified exploration from the specified user
+    if one exists.
+
+    Args:
+        user_id: str. The id of the user.
+        exploration_id: str. The specified exploration.
+
     Returns:
-        An integer 1-5, or None if there is no rating of this exploration by
+        An integer between 1 and 5 inclusive, or None if there is no
```

Comment:
int or None. An integer between 1 and 5 inclusive, or None if the user has not previously rated the exploration.